README.md

```diff
@@ -14,8 +15,38 @@ by leveraging the modern web.
 ## API Spec
 The preferred JSON object to be returned by the API should be structured as follows:
 
-### Users (for authentication)
+## Links to the Mock up Design
+1. ### Sign Up, Log In and Create Article
+[Full deck link](https://wireframepro.mockflow.com/view/M15c07edf891b45f0dfaa1f5cc85c68c61547577768098#/page/D31452032c99b79d208bf34f194933048)
+
+2. ### Homepage, Landing page and Profile
+ [Full deck link](https://wireframepro.mockflow.com/view/M724c1af1281229c8914e2f1e3b9e2a201547621799851)
+
+**How should this be manually tested?**
+
+1. Install pip first using this command `sudo apt-get install python3-pip`
+1. Then install virtualenv using pip3 `sudo pip3 install virtualenv `
+1. Now create a virtual environment `virtualenv venv `
```

Comment:
Add a windows option on how to create a virtualenv and how to activate it

Reply:
Kindly go through the ReadMe again..it has a part on how to create and activate virtualenv